src/services/DealService.ts

```diff
@@ -182,6 +193,55 @@ export class DealService {
     return dealIds;
   }
 
+  private async observeDeals(): Promise<void> {
+    const dealsObservable = await this.dataSourceDeals.getDealsObservables(this.ethereumService.defaultAccountAddress, true);
```

Comment:
By passing `true` as the second argument we skip first emitions of all queries. In my opinion we should always skip the first, because `observeDeals` method is always triggered after `getDeals`, so the deals are "fresh".

If we would only skip first query emitions when triggering  `observeDeals` for the first time, then it wouldn't skip first on accountAddress change.